app/api-requests/rpa-api/cph-check.js

```diff
@@ -0,0 +1,30 @@
+const DoesNotHaveAnyValidCph = require('./does-not-have-any-valid-cph')
+
+const between = (x, min, max) => {
+  return x >= min && x <= max
+}
+
+const inEngland = (cphNumber) => {
+  return between(cphNumber.slice(0, 2), 1, 51)
```

Comment:
Can we add some of these magic numbers as constants, same with the 8000 & 9999 below

Reply:
done